<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-1</y>
      <width>681</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Ajouter client</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>461</width>
          <height>241</height>
         </rect>
        </property>
        <property name="title">
         <string>                                    Rentrez les informations du client</string>
        </property>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero de Telephone</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>70</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>100</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_numero">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>130</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>200</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_email">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>160</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Afficher client</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>441</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>Liste des clients</string>
        </property>
        <widget class="QTableView" name="tab_client">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>21</y>
           <width>411</width>
           <height>181</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>Modifier client</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_10">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>461</width>
          <height>241</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero de Telephone</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>70</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>100</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_numero_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>130</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter_2">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>200</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_email_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>160</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>Supprimer client</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>411</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>                               Donner l'identifiant du client a supprimer</string>
        </property>
        <widget class="QLineEdit" name="le_id_supp">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>80</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_supprimer">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>140</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Carte fidelite</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajouter carte</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>441</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter une carte fidelite</string>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Points fidelite</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant client</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_carte">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_type">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>70</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_pt">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>100</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_client">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>130</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter_carte">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>180</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Afficher carte</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>441</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>Liste des cartes fidelite</string>
        </property>
        <widget class="QTableView" name="tab_carte">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>21</y>
           <width>411</width>
           <height>181</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Ameliorer carte</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Supprimer carte</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>411</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>                               Donner l'identifiant de la carte a supprimer</string>
        </property>
        <widget class="QLineEdit" name="le_id_supp_carte">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>80</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_supprimer_carte">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>140</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
